Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Defining ruby version
+FROM ruby:2.6.6-alpine
+
+# Copy app and set working dir
+COPY . /application
```

Comment:
Technically it doesn't really matter but it would be nice if we had a default location at which we store apps in containers for consistency.Most API's just run under the root directory or `/app`. 
Happy for us to pick a location and stick with it, for example `/usr/src/app`?

Reply:
As per discussion from standup I've changed this to `/usr/src/app` 👌